tests/common.py

```diff
@@ -1064,17 +1064,30 @@ async def start_reauth_flow(
         data: dict[str, Any] | None = None,
     ) -> ConfigFlowResult:
         """Start a reauthentication flow."""
-        return await hass.config_entries.flow.async_init(
-            self.domain,
-            context={
-                "source": config_entries.SOURCE_REAUTH,
-                "entry_id": self.entry_id,
-                "title_placeholders": {"name": self.title},
-                "unique_id": self.unique_id,
-            }
-            | (context or {}),
-            data=self.data | (data or {}),
-        )
+        return await start_reauth_flow(hass, self, context, data)
+
+
+async def start_reauth_flow(
```

Comment:
I don't think this should be a public function
```suggestion
async def _start_reauth_flow(
```